packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -0,0 +1,672 @@
+
+One important attribute modified through PATCH is the `active` attribute, representing the user’s current status.
+
+In order to deactivate a user, you need to enable the functionality in Okta. In the Okta Admin Console, under the **Provisioning** tab, click **To App** and select **Deactivate Users**.
+
+When a user is deactivated, Okta sends this request:
+
+    PATCH /scim/v1/users/48e0a2da-0999-4f2c-87f4-80432cfe6617 HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "id": "48e0a2da-0999-4f2c-87f4-80432cfe6617",
+        "active": false
+    }
+
+The response from the SCIM server needs to be the user’s SCIM profile:
+
+    HTTP/1.1 200 OK
+    Date: Fri, 18 Oct 2019 07:00:47 GMT
+    Content-Type: text/json;charset=UTF-8
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "id": "48e0a2da-0999-4f2c-87f4-80432cfe6617",
+        "userName": "test.user@okta.local",
+        "externalId": "00unrudb40SOK2eup0h7",
+        "name": {
+            "givenName": "Test1",
+            "familyName": "User"
+        },
+        "displayName": "Test1 User",
+        "locale": "en_US",
+        "active": false,
+        "emails": [{
+            "primary": true,
+            "value": "test.user@okta.local",
+            "type": "work"
+        }],
+        "groups": [],
+        "meta": {
+            "created": "2019-10-18T06:50:23+00:00",
+            "lastModified": "2019-10-18T07:00:47+00:00",
+            "version": "6feac5cd8e2e3a617037e2543e3e83d0"
+        }
+    }
+
+### Deleting specific users
+
+<ApiOperation method="delete" url="/users/${userId}" />
+
+Okta does not perform a DELETE operation on `/users/${userId}`.
+
+If a user is suspended, deactivated, or removed from the application in Okta, then Okta sends a PATCH request to set the `active` attribute to `false`.
+
+## Group operations
+
+### Creating groups
+
+<ApiOperation method="post" url="/groups" />
+
+To create a group in the SCIM server, an Okta administrator must push the group using the Okta Admin Console. In the Okta Admin Console, select the SCIM application from your list of applications. In the **Push Groups** tab, click **Push Groups**. For more information, see the [Using Group Push topic](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Directory_Using_Group_Push) in the Okta Help Center.
+
+After the group is selected, Okta makes a POST request to the service provider:
+
+    POST /scim/v1/groups HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "displayName": "Test SCIM2"
+    }
+
+When it receives this request the SCIM server responds with the group details, as it would for a GET operation to the `/groups/${groupId}/`:
```

Comment:
When it receives this request, the SCIM server responds with the group details as it would for a GET operation to the `/groups/${groupId}/`: